### PR TITLE
refactor(time-picker): remove redundant styles

### PR DESCRIPTION
**Pull request by SisIvanova**
Related to IgniteUI/igniteui-theming/issues/344

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [x] This PR contains breaking changes
 - [x] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Review by desig9stein — DISMISSED** [review]

**Comment by imincheva** [comment]
In Bootstrap the header title has 8px bottom margin, which should be removed.

File: projects/igniteui-angular/migrations/migration-collection.json [hunk 1]
```diff
@@ -201,6 +201,11 @@
             "version": "18.2.0",
             "description": "Updates Ignite UI for Angular from v18.1.x to v18.2.0",
             "factory": "./update-18_2_0"
+        },
+        "migration-41": {
+            "version": "18.2.2",
```
Comment (simeonoff) [inline projects/igniteui-angular/migrations/migration-collection.json:206]:
```suggestion
            "version": "18.2.3",
```

File: projects/igniteui-angular/migrations/migration-collection.json [hunk 2]
```diff
@@ -201,6 +201,11 @@
...
+            "description": "Updates Ignite UI for Angular from v18.2.0 to v18.2.2",
```
Comment (simeonoff) [inline projects/igniteui-angular/migrations/migration-collection.json:207]:
```suggestion
            "description": "Updates Ignite UI for Angular from v18.2.0 to v18.2.3",
```

File: projects/igniteui-angular/migrations/migration-collection.json [hunk 3]
```diff
@@ -201,6 +201,11 @@
...
+            "factory": "./update-18_2_2"
```
Comment (simeonoff) [inline projects/igniteui-angular/migrations/migration-collection.json:208]:
```suggestion
            "factory": "./update-18_2_3"
```

File: projects/igniteui-angular/migrations/update-18_2_2/index.spec.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import * as path from 'path';
+
+import { SchematicTestRunner, UnitTestTree } from '@angular-devkit/schematics/testing';
+import { setupTestTree } from '../common/setup.spec';
+
+const version = '18.2.2';
```
Comment (simeonoff) [inline projects/igniteui-angular/migrations/update-18_2_2/index.spec.ts:6]:
```suggestion
const version = '18.2.3';
```

File: projects/igniteui-angular/migrations/update-18_2_2/index.ts [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import type {
+    Rule,
+    SchematicContext,
+    Tree
+} from '@angular-devkit/schematics';
+import { UpdateChanges } from '../common/UpdateChanges';
+
+const version = '18.2.2';
```
Comment (simeonoff) [inline projects/igniteui-angular/migrations/update-18_2_2/index.ts:8]:
```suggestion
const version = '18.2.3';
```

**Review by simeonoff — APPROVED** [review]

**Pull request merged**